pkgs/development/compilers/rustc/bootstrap.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchurl }:
+
+let
+  platform = if stdenv.system == "i686-linux"
+    then "linux-i386"
+    else if stdenv.system == "x86_64-linux"
+    then "linux-x86_64"
+    else if stdenv.system == "i686-darwin"
+    then "macos-i386"
+    else if stdenv.system == "x86_64-darwin"
+    then "macos-x86_64"
+    else abort "no snapshot to bootstrap for this platform (missing platform url suffix)";
+
+  snapshotHashLinux686 = "0e0e4448b80d0a12b75485795244bb3857a0a7ef";
```

Comment:
irrelevant since this stable is the last version that uses snapshots, so the information is outdated...